mousestyles/mww.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+
```

Comment:
should we have the 
`from __future__ import print_function, division, absolute_import` for python 3 support?, also I think the standard is import itertools first, then numpy, scipy, and other after